layers/+tools/shell/packages.el

```diff
@@ -206,9 +206,8 @@
 
 (defun shell/post-init-projectile ()
   (spacemacs/set-leader-keys
-    "p'" 'spacemacs/projectile-shell-pop
-    "p$t" 'projectile-multi-term-in-root)
-  (spacemacs/declare-prefix "p$" "projects/shell"))
```

Comment:
Do you accidentally removed `(spacemacs/declare-prefix "p$" "projects/shell")`

Reply:
This was intentional:`p$t` was the only leader key under prefix `p$`, so now that we have a command which uses the default shell (whether multi-term or other) so I removed this prefix and made leader keys `p$` run `spacemacs/projectile-shell`.

Reply:
Oh right.